src/github.com/getlantern/lantern/feed.go

```diff
@@ -0,0 +1,170 @@
+package lantern
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"math"
+	"net/http"
+	"strings"
+
+	"github.com/getlantern/eventual"
+	"github.com/getlantern/flashlight/util"
```

Comment:
If you put feed into its own package and remove the dependency on flashlight/util, that makes this a nicely isolated package.